### PR TITLE
Update documentation to be clearer

### PR DESCRIPTION
**Pull request by Sgtpluck**
## Changes proposed in this pull request:
- Updated two pages of documentation that I found confusing while implementing uaa authentication

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/Sgtpluck:dmm/uaa-update)

## Security Considerations
[Note the any security considerations here, or make note of why there are none]
none! it's just a documentation update

### CODE REVIEW

File: _docs/management/leveraging-authentication.md [hunk 1]
```diff
@@ -92,13 +92,14 @@ access token. Here is where things get fun.
 
 3.  The `access_token` is a JSON Web Token that can be decoded using a
     library such as [node-jsonwebtoken](https://github.com/auth0/node-jsonwebtoken).
-    See https://jwt.io/ for a list of libraries for various languages. Decode it
-    to get the authenticated user's `email`, which you can then use within
+    See https://jwt.io/ for a list of libraries for various languages. 
+    
+    Verify the token's signature using cloud.gov's JWK Set and the RSA256 alogrithm. This step ensures the
+    token is authentic. The JWK Set for cloud.gov's UAA is located at `https://uaa.fr.cloud.gov/token_keys`.
```
Comment (Sgtpluck) [inline _docs/management/leveraging-authentication.md:98]: I wanted to make this seem less like an optional step, and also make the language more google-able for those of use who are new to implementing authentication!

File: _docs/services/cloud-gov-identity-provider.md [hunk 2]
```diff
@@ -23,24 +23,42 @@ Plan Name | Description |
 To create a service instance that can provision identity provider credentials, run the following command:
 
 ```sh
-cf create-service cloud-gov-identity-provider oauth-client my-uaa-client
+cf create-service cloud-gov-identity-provider oauth-client <SERVICE_INSTANCE_NAME>
```
Comment (Sgtpluck) [inline _docs/services/cloud-gov-identity-provider.md:26]: i was trying to make it more clear here that `my-uaa-client` was something user-provided and not part of the command.

**Comment by bengerman13** [comment]
Thanks @Sgtpluck, these changes look great. 
Our merge checks only run against branches from maintainers, so I'm going to pull this into a new branch and PR from there.

**Comment by Sgtpluck** [comment]
Merged via https://github.com/cloud-gov/cg-site/pull/2061

**Pull request closed without merging**